kaggle-classification/keras_trainer/model.py

```diff
@@ -42,11 +42,11 @@
     learning_rate=0.00005,
     dropout_rate=0.5,
     batch_size=128,
-    epochs=10,
+    epochs=1,
     sequence_length=250,
     embedding_dim=100,
     train_embedding=False,
-    model_type='single_layer_cnn',
+    model_type='cnn_with_attention',
```

Comment:
I think we should leave the basic model as the default unless there's a good reason to do otherwise.